content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,125 @@
+# plots modify
+
+Modify display properties of [plot metrics](/doc/command-reference/plots) files.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t <path>] [-x X] [-y Y]
+                        [--no-csv-header] [--title <text>] [--x-label <text>]
+                        [--y-label <text>] [--unset [<prop> [<prop> ...]]]
+
+positional arguments:
+  target                Plot file to set props to.
+```
+
+## Description
+
+It might be not convenient for users or automation systems to specify all the
+_visualization properties_ (such as `y-axis`, `template`, `title`, etc.) each
+time plots are generated. This command sets (or unsets) default visualization
+props for specific plot files. These properties affect the remaining
+subcommands: `dvc plots show` and `dvc plots diff`.
+
+The `target` plot file should be a plot output of one of the
+[DVC pipeline](/doc/command-reference/pipeline) stages (see the `--plots` option
+of `dvc run`) listed in the
+[`dvc.yaml`](/doc/user-guide/dvc-files-and-directories) file. `dvc plots modify`
+adds the options to `dvc.yaml`.
```

Comment:
Is this also the case now for plots show/diff? Those refs currently say they can use any metrics file from Git, DVC, or untracked. Cc @pared

Reply:
Yeah, I can confirm all subcommands require metrics files that have been registered with dvc run --plots before... Will update ⏳

Reply:
Updated.